### PR TITLE
Add monitoring tools from utils node

### PR DESCRIPTION
**Pull request by apdibbo**
Adds a script which calculates the number of instances of each flavor can be created
Openstack service status into influx
QUota changes into influx

### CODE REVIEW

File: MonitoringTools/usr/local/bin/limits-to-influx.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+parser = SafeConfigParser()
+try:
+   parser.read('/etc/influxdb.conf')
+   host = parser.get('db', 'host')
+   database = parser.get('db', 'database')
+   username = parser.get('auth', 'username')
+   password = parser.get('auth', 'password')
+   instance = parser.get('cloud','instance')
+except:
+   print 'Unable to read from config file'
```
Comment (DavidFair) [inline MonitoringTools/usr/local/bin/limits-to-influx.py:21]: This file needs `2to3` treatment too

File: MonitoringTools/usr/local/bin/slottifier.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+   instance = parser.get('cloud','instance')
+except:
+   print 'Unable to read from config file'
+   sys.exit(1)
+
+url = 'http://'+host+'/write?db='+database +'&precision=s'
+
+env = os.environ.copy()
+
+env = os.environ.copy()
```
Comment (DavidFair) [inline MonitoringTools/usr/local/bin/slottifier.py:30]: Duplicated LOC

File: MonitoringTools/usr/local/bin/slottifier.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+   host = parser.get('db', 'host')
+   database = parser.get('db', 'database')
+   username = parser.get('auth', 'username')
+   password = parser.get('auth', 'password')
+   instance = parser.get('cloud','instance')
+except:
+   print 'Unable to read from config file'
+   sys.exit(1)
+
+url = 'http://'+host+'/write?db='+database +'&precision=s'
```
Comment (DavidFair) [inline MonitoringTools/usr/local/bin/slottifier.py:26]: We have these LOC 4 times across multiple files, IMO this is worth moving into another file and have the other scripts import it so we only have to change it once

**Review by DavidFair — CHANGES_REQUESTED** [review]
Partial review

**Comment by apdibbo** [comment]
How does one 2to3?

**Comment by apdibbo** [comment]
I am actually going to make another quick change to this

**Comment by anish-mudaraddi** [comment]
see #127 for refactored version of this code

**Pull request closed without merging**